lib/rafinesque/initializer.rb

```diff
@@ -0,0 +1,4 @@
+# Rafinesque initializer
+
+require 'rafinesque/i18n_backend'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.